logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/mapper/CallMapper.kt

```diff
@@ -144,14 +145,15 @@ class CallMapperImpl : CallMapper {
         CallStatus.CLOSED -> CallEntity.Status.CLOSED
     }
 
-    private fun toCallStatus(callStatus: CallEntity.Status): CallStatus = when (callStatus) {
+    override fun toCallStatus(callStatus: CallEntity.Status?): CallStatus = when (callStatus) {
         CallEntity.Status.STARTED -> CallStatus.STARTED
         CallEntity.Status.INCOMING -> CallStatus.INCOMING
         CallEntity.Status.MISSED -> CallStatus.MISSED
         CallEntity.Status.ANSWERED -> CallStatus.ANSWERED
         CallEntity.Status.ESTABLISHED -> CallStatus.ESTABLISHED
         CallEntity.Status.STILL_ONGOING -> CallStatus.STILL_ONGOING
         CallEntity.Status.CLOSED -> CallStatus.CLOSED
+        else -> CallStatus.CLOSED
```

Comment:
are we sure we want to map it to CLOSED on else ? it could sneak in into our code that we forget to map something in the future if we add anything to CallEntity.Status

Reply:
Totally agree, this is why I modified the mappers to decide and choose a treatment for them.